content/22.vaccines-trad.md

```diff
@@ -458,6 +459,22 @@ In general, deciding on the prioritization and allocation of the COVID-19 vaccin
+While some of these challenges in vaccine rollout in low- and middle-income countries are being supported through COVAX, [@url:https://www.unicef.org/supply/covax-ensuring-global-equitable-access-covid-19-vaccines], many issues persist worldwide [@doi:10.1016/S0140-6736(22)00328-2;@doi:10.1093/cid/ciac361; @doi:10.1080/20477724.2022.2091862].
+COVAX also failed to distribute its promised 2 billion vaccine doses on time due to multiple complications [@url:https://www.washingtonpost.com/world/2021/12/10/covax-doses-delivered/].
+
+Another major challenge to global vaccine distribution is vaccine hesitancy, which the WHO has designated as a leading global health threat [@url:https://www.who.int/news-room/spotlight/ten-threats-to-global-health-in-2019]. 
+Polling in the U.S. in January (2021) suggested that 20% of individuals were reluctant to receive a vaccine in January 2021 with a further 31% expressing some hesitancy to a lesser extent [@url:kff-covid-19-vaccine-monitor-january-2021-vaccine-hesitancy; @doi:10.1056/NEJMms2101220].
+A survey of 8,243 long-term healthcare workers in November 2020 (Indiana, USA) reported that only 69% of respondents would ever consider receiving an FDA-approved vaccine due to their perceived risk of side effects (70%), health concerns (34%) efficacy (20%), and religious beliefs (12%) [@doi:10.1111/jgs.17022].
+Notably, almost a third of parents surveyed in the United States in March 2021 expressed concerns about vaccinating their children against COVID-19 [@doi: 10.1177/003335492211143].
+Indeed, vaccine hesitancy has been reported as a significant barrier to vaccine distribution sin countries of North and South America, Europe, Asia, and Africa [@doi:10.1002/jmv.28156; @doi:10.1126/sciadv.abm9825; @doi:10.1186/s41256-022-00255-1; @doi:10.1080/20477724.2021.2011580; @doi:10.1136/bmjopen-2021-052432]. 
+Various factors have been associated with increased vaccine hesitancy including access to compelling misinformation via social media [@doi:10.1093/pnasnexus/pgac207; @doi:10.1038/s41591-022-01728-z], religious and conservative polictical beliefs [@doi:10.1371/journal.pone.0268926; @doi:10.1093/pubmed/fdab122; @doi:10.1177/1075547020960463; @doi:10.1377/hlthaff.2020.02254], and safety and efficacy concerns to highlight a few [@doi:10.1177/003335492211143].
+Many of the concerns regarding safety and efficacy have mostly focused on the novel mRNA technologies due to their perceived speed of development and expidited clinical trial process [@doi:10.1186/s13037-021-00291-9], however, general vaccine hesistancy relating to traditional vaccine platforms existed long before the pandemic and the distribution of the novel mRNA vaccines [@doi:10.1016/j.vaccine.2015.09.035; @doi:10.1016/j.mayocp.2015.09.006].
 
```

Comment:
Maybe I should pull the data about how many Americans have actually received Novavax and add a sentence here? That was allegedly supposed to be a game changer for vaccine skeptics because it wasn't a new technology, but I definitely haven't heard anything suggesting that it has been.

Reply:
That's a good idea actually, I took the low hanging fruit of mRNA vaccines to address that comment

Reply:
```suggestion
While in the United States, it was hoped that Novavax's Nuvaxovid would appeal to the vaccine hesitant [@url:https://www.ama-assn.org/delivering-care/public-health/what-sets-apart-novavax-option-other-covid-19-vaccines; @url:https://www.galvnews.com/health/free/article_8ddebef4-1db0-579a-b0d2-852f2031d881.html], but this protein subunit vaccine has not led to the uptake hoped [@doi:10.1001/jama.2022.13661; @url:https://www.cnbc.com/2022/07/15/why-new-novavax-covid-vaccine-wont-win-over-unvaccinated-americans.html]. 
```

Reply:
![novavax](https://user-images.githubusercontent.com/8975841/208579242-2230cc4a-9825-445b-aa76-548d0205c2e0.png)
Hey @RLordan, I wasn't able to get a good visualization of this data. Things are being buggy and I'm not sure if it's the dataset or the plotting package. I can probably get something like this to work if we think it would be useful, but also maybe the sentence I suggested above is enough!

Reply:
I think a plot like this would be useful. People respond to visuals more. Certainly, it shows that J&J and Novavax never kicked off that much once approved.